src/Graphics/Vty/PictureToSpans.hs

```diff
@@ -309,7 +309,7 @@ addMaybeClippedJoin name skip remaining offset i0Dim i0 i1 size = do
     when (state^.remaining <= 0) $ fail $ name ++ " with remaining <= 0"
     case state^.skip of
         s -- TODO: check if clipped in other dim. if not use addUnclipped
-          | s >= size -> fail $ name ++ " on fully clipped"
+          | s > size -> fail $ name ++ " on fully clipped"
```

Comment:
So it seems this was an issue with empty images because s was 0 and size was 0, the minimal change from `>=` to `>` seems to have solved that case and *I think* should still catch any actual error cases.